archive/p/python/prime-number.py

```diff
@@ -4,9 +4,7 @@
 
 
 def is_prime(x):
-    if x <= 1:
-        return True
-    if x % 2 == 0:
+    if (x % 2 == 0 and x is not 2) or (x == 1):
```

Comment:
It appears these changes break the solution:

```
0 -> Usage: please enter non-negative value
1 -> False
2 -> True
3 -> True
4 -> False
```

Reply:
Nevermind! My brain went blank here. This totally works.

Reply:
That said, we exclude 0 which doesn't match the usage statement.

Reply:
ahh ok i'll fix the 0.